packages/firebase/src/index.rn.ts

```diff
@@ -24,6 +24,7 @@ import '../database';
 //  `atob`). We should provide a RN build that works out of the box.
 import '../storage';
 import '../firestore';
+import '../firestore/bundle';
```

Comment:
The React Nativce build of the main Firestore package splits up bundles and non-bundled code. I think we should do the same here.

Reply:
This is for "complete" build, which include everything (except memory only builds) on other platforms.